src/primo/browzine-primo-adapter.js

```diff
@@ -445,7 +445,7 @@ browzine.primo = (function() {
 
   function directToPDFTemplate(directToPDFUrl) {
     var pdfIcon = "https://assets.thirdiron.com/images/integrations/browzine-pdf-download-icon.svg";
-    var articlePDFDownloadLinkText = browzine.articlePDFDownloadLinkText || browzine.primoArticlePDFDownloadLinkText  || "Download Now";
+    var articlePDFDownloadLinkText = browzine.articlePDFDownloadLinkText || browzine.primoArticlePDFDownloadLinkText  || "Download PDF";
```

Comment:
I didn't notice this default text being changed anywhere - had John mentioned wanting this somewhere Tian?

Reply:
Pretty sure it was a copy-and-paste issue. Summon text looks like this: `Article PDF: Download Now` and that's where I believe this comes from. The `README` default text is "Download PDF": https://github.com/thirdiron/browzine-discovery-service-adapters/blob/develop/README.md#primo-adapter-customizations